content/ember-data-polaris-2023-04-11.md

```diff
@@ -0,0 +1,73 @@
+---
+title: EmberData Polaris Update
+authors:
+  - chris-thoburn
```

Comment:
You'll want to add more authors here as appropriate, or change the "we" statements throughout the article.